### DISCUSSION TITLE
I very much like Unstoppable Wallet, however, there are one or two obvious coins missing

### DISCUSSION THREAD

**Discussion by xcelente** [horizontalsystems/unstoppable-wallet-android#6719] [Ideas]
Hello,

I very much like Unstoppable Wallet, however, there are one or two obvious coins missing that I find in other multi-coin wallets.

First up, I would love to see Decred added, and why not, it's an OG and still going strong. I think the development is stronger than ever despite the rank and price. Just take a look and tell me I'm wrong.

Next up, similar to other person's post here, I would definitely like to see more "privacy coins". I say privacy coins but, really, they are just highly fungible coins, the way an electronic cash system should be tbh. I mean to say, this is the point of me having Unstoppable Wallet on my phone, because it's a hot wallet with spending money, not like I will put my life's savings on it. Spending is my motivation, relying less on banks. This wallet is one of the wallets I show others when I am on-boarding them for the first time. And I definitely would prefer to be spending highly fungible coins with others. My favorite privacy coins, in order of preference, are beam, grin and monero (dash is already on it and I'm thankful).

Cheers

**Comment by serrq** [comment]
Monero integration is on the way (0.37) if there aren't particular issues.

**Reply by xcelente** [reply]
Thanks for the update regarding monero. I hope either beam or grin can also be considered as they represent a novel way to transact that no other do.

Btw, if there are any implementation issues to add decred, please let us know as it somewhat baffles me why not. Trust Wallet has decred but I would prefer to run just one multi-coin wallet.
